docs/archives.md

```diff
@@ -61,14 +61,14 @@ export default withArchive(
 )( TodayArchive );
 ```
 
-`withArchive` components will automatically load the archive if it's not available in the store. Your component receives seven props:
+`withArchive` components will automatically load the archive if it's not available in the store. Your component receives five data props, and two action props:
```

Comment:
this seems like a pain to keep in sync!

Reply:
The API should be pretty stable (we don't want to keep adding new props, since they might conflict), I just wanted to separate the terminology for clarity in the new options. :)